dashboard/app/views/levels/editors/_mini_rubric.html.haml

```diff
@@ -1,7 +1,7 @@
 %legend.control-legend{data: {toggle: "collapse", target: "#mini_rubric"}}
   Mini Rubric
 
-#mini_rubric.in.collapse
```

Comment:
out of curiosity, what was `.in` for?

Reply:
it starts the collapser area as open instead of closed